@stellar/lyra-api/src/index.ts

```diff
@@ -3,3 +3,4 @@ import { signTransaction } from "./signTransaction";
 import { isConnected } from "./isConnected";
 
 export { getPublicKey, signTransaction, isConnected };
+export default { getPublicKey, signTransaction, isConnected };
```

Comment:
started testing on `lab` and realized only using named exports was a little annoying. importing a function called `isConnected` in a big app is weird bc it's not descriptive as to what it is and could lead to naming collisions.

In lab, what i really wanted is to be able to call `lyraApi.isConnected()`. My thought is to take the React route and export as both default and as named

Reply:
Nice, that makes sense to me